lua/octo/pickers/fzf-lua/pickers/users.lua

```diff
@@ -1,118 +1,199 @@
 local entry_maker = require "octo.pickers.fzf-lua.entry_maker"
+local octo_config = require "octo.config"
+local queries = require "octo.gh.queries"
 local fzf = require "fzf-lua"
 local gh = require "octo.gh"
 local graphql = require "octo.gh.graphql"
 local picker_utils = require "octo.pickers.fzf-lua.pickers.utils"
 local utils = require "octo.utils"
 
-return function(cb)
-  local formatted_users = {}
+local orgs = {}
```

Comment:
should this be scoped here or defined within function?

Reply:
I use if for the inception picker of orgs.
if its an org, I need to open another picker with the org teams, so I keep the orgs on the module level so I can access it another time.
Basically this is the same thing that was before my PR, but now just for orgs and not for users.
for users I can keep everything I need in the results table and use `with-nth` option of fzf to not show the irrelevant data.